tests/functionsTest.php

```diff
@@ -30,21 +30,30 @@ function testSuccessCreateRecipeCards2()
 
     function testMalformedCreateRecipeCards()
     {
-        $inputa = "hello";
+        $inputa = 'hello';
 
         $this->expectException(TypeError::class);
 
         $case = createRecipeCards($inputa);
     }
 
-    function testSuccessLinkValidation()
+    function testSuccessValidateString()
```

Comment:
I think the functions need the public prefix

Reply:
I don't think so for unit testing? At least not in my notes - may wait and see if Ash says anything